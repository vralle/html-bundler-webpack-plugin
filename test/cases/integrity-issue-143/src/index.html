<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <link href="@src/style.css" rel="stylesheet">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
</html>