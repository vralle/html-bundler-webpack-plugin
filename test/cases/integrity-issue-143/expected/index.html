<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
    <link href="css/style.7947baa4.css" rel="stylesheet" integrity="sha384-04yHBDae3wtc7se23yI4Zf97CVcVzENwwbsMzwsUyF3/Olso6e41tHR9ZxSZayPY" crossorigin="anonymous">
  </head>
  <body>
    <h1>Hello World!</h1>
  </body>
  <script src="main.js" integrity="sha384-i658BCTWidoV7NM7d2CKm6hpm2l17+pcg4lq7rMo4IWen2DRdysBWBlEXNx1R0aL" crossorigin="anonymous"></script>
</html>